silkrpc/txpool/transaction_pool.hpp

```diff
@@ -215,6 +249,33 @@ class TransactionPool final {
         co_return status_info;
     }
 
+    asio::awaitable<TxPoolTransactions> get_transactions() {
+        TxPoolTransactions txpool_transactions{};
+        const auto start_time = clock_time::now();
+        SILKRPC_DEBUG << "TransactionPool::get_transactions\n";
+        ::txpool::AllRequest request{};
+        AllAwaitable all_awaitable{executor_, stub_, queue_};
+        const auto reply = co_await all_awaitable.async_call(request, asio::use_awaitable);
+        const auto txs_size = reply.txs_size();
+        for (int i = 0; i < txs_size; i++) {
+           struct ElementInfo element{};
+           const auto tx = reply.txs(i);
+           const auto rlptx = tx.rlptx();
+           const auto tx_sender = tx.sender();
+           const auto address_bytes = silkworm::from_hex(tx.sender());
+           element.sender  = silkworm::to_evmc_address(address_bytes.value());
+           element.rlp = silkworm::Bytes{rlptx.begin(), rlptx.end()};
+           if (tx.type() == Txpool::AllReply_Type_PENDING)
```

Comment:
Please use curly braces also for one-line `if`, `else if`, `else`

Reply:
ok